### PR TITLE
Move the policy HTML generation to a template.

### PR DESCRIPTION
**Pull request by kberg**
I think this is working.

image
image

### CODE REVIEW

File: src/client/components/turmoil/Agendas.vue [hunk 1]
```diff
@@ -0,0 +1,209 @@
+<template>
+  <div :class="getClass">
+    <template v-if="id === 'mb01'">
+      <div class="resource money party-resource">1</div> /
+      <div class="resource-tag tag-building party-resource-tag"></div>
+    </template>
+    <template v-if="id === 'mb02'">
```
Comment (bafolts) [inline src/client/components/turmoil/Agendas.vue:7]: Can this make use of `v-else-if`?

File: src/client/components/Turmoil.vue [hunk 2]
```diff
@@ -238,68 +99,45 @@ export default Vue.extend({
...
-          break;
+          return politicalAgendas.greens.policyId;
         }
       }
-
-      if (policyId !== undefined) {
-        const policy = AGENDA_HTML.get(policyId);
-        return policy || `No ruling Policy`;
-      }
-      return '<p>' + this.$t('No ruling Policy') + '</p>';
```
Comment (bafolts) [inline src/client/components/Turmoil.vue:302]: Did this get lost? I don't see if in the templates. It should be in a final `v-else`.

File: src/client/components/Turmoil.vue [hunk 3]
```diff
@@ -238,68 +99,45 @@ export default Vue.extend({
...
-          bonusId = politicalAgendas.greens.bonusId;
-          break;
+          return politicalAgendas.greens.bonusId;
         }
       }
-      if (bonusId !== undefined) {
-        const bonus = AGENDA_HTML.get(bonusId);
-        return bonus || `No ruling Bonus`;
-      }
-      return `No ruling Bonus`;
```
Comment (bafolts) [inline src/client/components/Turmoil.vue:270]: Did this go missing or do we not need it?

File: src/client/components/turmoil/Agendas.vue (hunk 1 above)
Comment (kberg) [reply src/client/components/turmoil/Agendas.vue:7]: It could, though it's kind of valueless here.

File: src/client/components/Turmoil.vue (hunk 2 above)
Comment (kberg) [reply src/client/components/Turmoil.vue:302]: It won't be possible. The type is now tightly defined.

File: src/client/components/Turmoil.vue (hunk 3 above)
Comment (kberg) [reply src/client/components/Turmoil.vue:270]: Same -- we don't need it.

File: src/client/components/turmoil/Agendas.vue (hunk 1 above)
Comment (bafolts) [reply src/client/components/turmoil/Agendas.vue:7]: The value would only be for performance. With this `v-if` I believe `vue` will have to check each `id` even after it gets a match. With `v-else-if` once a match is reached `vue` doesn't have to compare the rest. Could also let reader know this is used like a switch statement.

**Review by bafolts — APPROVED** [review]

File: src/client/components/turmoil/Agendas.vue (hunk 1 above)
Comment (kberg) [reply src/client/components/turmoil/Agendas.vue]: Done. Reasonable. Vue.js, where's my `v-switch`?

File: src/client/components/Turmoil.vue (hunk 2 above)
Comment (kberg) [reply src/client/components/Turmoil.vue:302]: Ah the heck with it. I'm being silly not adding a default failure condition.

**Pull request merged**